invenio_app_rdm/records_ui/templates/semantic-ui/invenio_app_rdm/records/details/description.html

```diff
@@ -7,9 +7,13 @@
   it under the terms of the MIT License; see LICENSE file for more details.
 #}
 
+{%- from "invenio_app_rdm/records/macros/detail.html" import show_add_descriptions %}
+
 {% set description = metadata.get('description') %}
 {% if description %}
+<h2 id="description-heading">{{ _('Description') }}</h2>
 {# description data is being sanitized by marshmallow in the backend #}
-<h2 class="ui header">{{ _('Description')}}</h2>
-<div>{{ description | safe }}</div>
+{{ description | safe }}
 {% endif %}
+
+{{ show_add_descriptions(record.ui.additional_descriptions) if record.ui.additional_descriptions }}
```

Comment:
![Screenshot 2022-02-03 at 13 53 54](https://user-images.githubusercontent.com/1698163/152362502-7f7cbacd-f09b-4278-9339-7b2cb6179060.png)

Perhaps simply make them into sections like the description.